src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -152,7 +152,7 @@ public TestCase updateTestCase(
     if (measure.getTestCases() == null) {
       measure.setTestCases(new ArrayList<>());
     }
-
+    measureService.verifyAuthorization(username, measure);
```

Comment:
this may not work. you would want to fetch the measure using `measureService.findMeasureById()` line 146 
instead of  `Measure measure = findMeasureById(measureId);` use 
`Measure measure  = measureService.findMeasureById()`

Reply:
ah, sorry I missed that